lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -6,22 +6,19 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   """
   use Membrane.Bin
 
+  alias Membrane.ICE.TurnUtils
   alias Membrane.WebRTC.{SDP, EndpointBin}
   alias Membrane.WebRTC
   alias Membrane.RTC.Engine
 
   @type stun_server_t() :: ExLibnice.stun_server()
   @type turn_server_t() :: ExLibnice.relay_info()
 
-  @type extension_options_t() :: [
-          vad: boolean()
-        ]
-
-  @type packet_filters_t() :: %{
-          (encoding_name :: atom()) => [Membrane.RTP.SessionBin.packet_filter_t()]
-        }
-
-  def_options stun_servers: [
+  def_options id: [
+                spec: String.t(),
+                description: "Id of endpoint"
```

Comment:
There is no difference. It was added because `TurnUtils.generate_credentials` require a name and in the previous implementation, peer_id was used. So I decided that it should stay the same.